tools/travis/setup.sh

```diff
@@ -14,8 +14,11 @@ docker info
 
 # Python
 sudo apt-get -y install python-pip
+pip install --upgrade pip setuptools
 pip install --user jsonschema
 pip install --user couchdb
+pip install --user backports.ssl_match_hostname
```

Comment:
https://travis-ci.org/openwhisk/openwhisk/builds/208065811 has the line...

fatal: [172.17.0.1]: FAILED! => {"changed": false, "failed": true, "msg": "Failed to import docker-py - No module named ssl_match_hostname. Try `pip install docker-py`"}

This was probably related to the broken build this morning so `backports.ssl_match_hostname` might no longer be needed.